pages/sdk/[...slug].vue

```diff
@@ -73,6 +90,19 @@ const links = computed(() =>
         v-if="page.body"
         :value="page"
       />
+      <p
+        v-if="page.title === 'FAQ'"
+        class="mt-10"
+      >
+        <FaqAccordion />
+      </p>
+
+      <p
+        v-if="page.title === 'Glossary'"
+        class="mt-10"
+      >
+        <GlossaryAccordion />
+      </p>
```

Comment:
While the accordion does make things look succinct on the page, can we please remove them and just use a list of H2 headers? The content gets hidden away and the Table of Contents is empty, this makes it bad for accessibility.